src/components/safe-messages/MsgModal/Confirmations.tsx

```diff
@@ -0,0 +1,31 @@
+import { Box, Stack, Typography } from '@mui/material'
+import { SafeMessageStatus, type SafeMessage } from '@safe-global/safe-gateway-typescript-sdk'
+import EthHashInfo from '@/components/common/EthHashInfo'
+
+const Confirmations = ({ message, threshold }: { message?: Omit<SafeMessage, 'type'>; threshold: number }) => {
+  if (!message) return null
+
+  return (
+    <Box
+      mt={2}
+      borderRadius={1}
+      padding={1}
+      border={'1px solid'}
+      borderColor={message?.status === SafeMessageStatus.NEEDS_CONFIRMATION ? 'red' : 'green'}
+    >
+      <Typography fontWeight={700} mb={1}>
+        Confirmations{' '}
+        <Typography component="span" sx={{ color: ({ palette }) => palette.border.main }}>
+          ({`${message?.confirmationsSubmitted || 0} of ${threshold}`})
+        </Typography>
+      </Typography>
+      <Stack direction="column" spacing={2}>
+        {message?.confirmations?.map(({ owner }) => (
```

Comment:
Please don't reinvent the wheel and use the existing signatures component from tx details. It looks much nicer.

Reply:
I've tried with that component but it felt unnecessary big with the "Created" and Hide/Show options. it is a direct substitution tho. I can introduce it again

Reply:
Those could be toggled via a new prop.